src/Factories/PDO_Factory.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+class PDO_Factory
```

Comment:
Please use camel case here to be consistent with your other classes

Reply:
Done.